### PR TITLE
Feature/support dropdownmenu position fixed

### PR DESCRIPTION
**Pull request by sh11mo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://reactstrap.github.io/components/dropdowns/

### CODE REVIEW

File: types/reactstrap/reactstrap-tests.tsx [hunk 1]
```diff
@@ -4528,3 +4528,39 @@ function Example127() {
...
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
+
+  render() {
+    return (
+      <Dropdown
+        isOpen={this.state.dropdownOpen}
+        toggle={this.toggle}
+        positionFixed
```
Comment (kyletsang) [inline types/reactstrap/reactstrap-tests.tsx]: positionFixed should be on the DropdownMenu element

File: types/reactstrap/v7/reactstrap-tests.tsx [hunk 2]
```diff
@@ -4357,3 +4357,39 @@ function Example126() {
...
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
+
+  render() {
+    return (
+      <Dropdown
+        isOpen={this.state.dropdownOpen}
+        toggle={this.toggle}
+        positionFixed
```
Comment (kyletsang) [inline types/reactstrap/v7/reactstrap-tests.tsx]: positionFixed should be on the DropdownMenu element

**Review by kyletsang — CHANGES_REQUESTED** [review]
Looks good. A couple of minor changes are needed in the tests. The positionFixed bool was placed on the wrong element.

File: types/reactstrap/reactstrap-tests.tsx (hunk 1 above)
Comment (ichiwa) [reply types/reactstrap/reactstrap-tests.tsx]: Oh... thanks. I will fix it 👍

**Comment by ichiwa** [comment]
@kyletsang 
I fixed to use positionFixed correctly in unit test. PTAL.

**Review by kyletsang — APPROVED** [review]
Looks good!

**Comment by sheetalkamat** [comment]
@ichiwa please fix the merge conflicts

**Comment by ichiwa** [comment]
@sheetalkamat PTAL again!

**Pull request merged**